tsdb/agent/db.go

```diff
@@ -953,6 +953,11 @@ func (a *appender) UpdateMetadata(storage.SeriesRef, labels.Labels, metadata.Met
 	return 0, nil
 }
 
+// AppendCreatedTimestamp wasn't implemented for agent mode, yet.
+func (a *appender) AppendCreatedTimestamp(ref storage.SeriesRef, l labels.Labels, t int64) (storage.SeriesRef, error) {
+	return 0, nil
+}
```

Comment:
TODO: Open issue

Reply:
this should be resolved before merging imo

Reply:
Hm, I would challenge doing this in here.

One reason is that agent does not need CT in form of zero samples as it's mostly for querying. If we want that for remote write 1.0 users (stateful way, so not very useful), we can add that later.

Second reason is scope, why it's important to do this atomically?